pytripgui/treewidget_vc/treewidget_view.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QTreeWidgetItem
+
+logger = logging.getLogger(__name__)
+
+
+class TreeWidgetView:
+    def __init__(self, ui):
```

Comment:
at least some class description would be nice, like a quick outline of the design philosophy for the tree view (since they can be made in many ways) and/or what should there be in the treeview.

Reply:
I think in next month I will through it out and put QtTreeView instead of QtTreeWidget

Reply:
What is the advantage of QtTreeView?

Reply:
1. patient tree now works - it is written ugly, but I made a lesson, and want to reimplement this module.

2. I saw previously implementation, understand it and thought that was unnecessary too complex(Searching item in qtree and then deleting them means more code and potentially more problems). Now you just delete old structure and init everything - just use this same code which creates the tree. The problem was when you expand any of subtree - if you after this add plan (adding plan synchronize tree), so you will delete all structure, and then add everything from scratch. Then tree widget doesn't remember that you expand something. So I have to remember which one was expanded with a struct that contains tree (with PatientTreeModel class in patient_gui_model.py). So this approach also adds unnecessary complexity to code

3. qtreeview solves both problems presented in 3.

4. I think it will be easier, to reimplement it using qtreeview, and I want to get familiar with qtreeview :dancer:

Reply:
Thanks. When I wrote the code I was not sure which to choose. I experimented with both, and then ended up with QtTreeWidget. Because of NO DOCUMENTATION I cannot remember anymore why.

:stuck_out_tongue_winking_eye: